authz/providers/azure/rbac/rbac.go

```diff
@@ -30,9 +30,12 @@ import (
 
 	auth "go.kubeguard.dev/guard/auth/providers/azure"
 	"go.kubeguard.dev/guard/auth/providers/azure/graph"
+
 	"go.kubeguard.dev/guard/authz"
 	authzOpts "go.kubeguard.dev/guard/authz/providers/azure/options"
 
+	"go.kubeguard.dev/guard/util/httpclient"
```

Comment:
can you align this import with the rest of kubeguard imports?

Reply:
fixed